src/Microsoft.DotNet.Interactive/Kernel.cs

```diff
@@ -393,12 +393,22 @@ public void RegisterForDisposal(IDisposable disposable)
                                                 .Lines
                                                 .GetPosition(command.LinePosition);
 
+                var completions = GetDirectiveCompletionItems(
+                    directiveNode, 
+                    requestPosition);
+                
+                var upToCursor =
+                    directiveNode.Text[..command.LinePosition.Character];
+
+                var indexOfPreviousSpace =
+                    Math.Max(
+                        0, 
+                        upToCursor.LastIndexOf(" ", StringComparison.CurrentCultureIgnoreCase) + 1);
```

Comment:
Nit: `LastIndexOf()` returns `-1` on not found and since you're doing `+ 1` anyway the `Math.Max()` call is redundant, but it does help with the clarity.